src/sumologic-span-processor/find-span-context.ts

```diff
@@ -33,17 +32,15 @@ const findBestSpanInTime = (
     }
   }
 
-  // if not we will attach it to the span that ends no longer than some timeout
+  // if not found, try span where the longtask started
   for (let i = spans.length - 1; i >= 0; i -= 1) {
     const span = spans[i];
     if (
       span.instrumentationLibrary.name !== INSTRUMENTATION_LONG_TASK &&
-      !span.parentSpanId &&
```

Comment:
Why is this condition no longer necessary?

Reply:
We're still working on the best approach for guessing long task contexts.